wdl/GermlineCNVCase.wdl

```diff
@@ -227,7 +227,7 @@ task DetermineGermlineContigPloidyCaseMode {
     RuntimeAttr runtime_attr = select_first([runtime_attr_override, default_attr])
 
     Float mem_gb = select_first([runtime_attr.mem_gb, default_attr.mem_gb])
-    Int command_mem_mb = ceil(mem_gb * 1000 - 500)
+    Int command_mem_mb = ceil(mem_gb * 1000 * 0.6)
```

Comment:
You can also do `ceil(mem_gb * 600)`.

Reply:
Yeah that would be shorter, but I'd prefer to leave it this way since IMO it's a little clearer where the factor of 1000 comes from.